spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/prometheus/PrometheusProperties.java

```diff
@@ -116,6 +118,21 @@ public Pushgateway getPushgateway() {
 		 */
 		private Map<String, String> groupingKey = new HashMap<>();
 
+		/**
+		 * Enable publishing via a Prometheus Pushgateway with Basic Auth.
+		 */
+		private Boolean authEnabled = false;
+
+		/**
+		 * Basic Auth Username.
```

Comment:
I am not sure if "basic auth" is really important here.

Reply:
pushgateway already support basic auth, we just need to pass auth parameters to `setConnectionFactory` function.

Reply:
I am aware of that. The comment is on the key description.

Reply:
ok